cryptol-remote-api/run_rpc_tests.sh

```diff
@@ -10,27 +10,22 @@ cabal v2-build exe:cryptol-eval-server
 pushd $DIR/python
 
 NUM_FAILS=0
+function run_test {
+    "$@"
+    if (( $? != 0 )); then NUM_FAILS=$(($NUM_FAILS+1)); fi
+}
 
 echo "Setting up python environment for remote server clients..."
-python3 -m venv virtenv
-. virtenv/bin/activate
-pip install -r requirements.txt
+poetry install
```

Comment:
CI is failing due to not being able to `poetry`, so you'll most likely need to add a GitHub Actions step to install it along [these lines](https://github.com/GaloisInc/saw-script/blob/2d92f2a9ef91d65f58ecb6679f34a5528b160b4e/.github/workflows/ci.yml#L208-L210).